tests/integration/kubernetes/k8s-confidential-attestation.bats

```diff
@@ -90,11 +90,5 @@ teardown() {
 		skip "Test skipped as KBS not setup"
 	fi
 
-	[ -n "${pod_name:-}" ] && kubectl describe "pod/${pod_name}" || true
-	[ -n "${pod_config_dir:-}" ] && kubectl delete -f "${K8S_TEST_YAML}" || true
-
-	if [[ -n "${node_start_time:-}" && -z "$BATS_TEST_COMPLETED" ]]; then
-		echo "DEBUG: system logs of node '$node' since test start time ($node_start_time)"
-		exec_host "${node}" journalctl -x -t "kata" --since '"'$node_start_time'"' || true
-	fi
+	teardown_common "${node}" "${node_start_time:-}"
```

Comment:
Are `${node}` `${node_start_time}` not available to tests_common.sh already as IIRC they were defined in that files and it's loaded in?

Reply:
yep, https://github.com/kata-containers/kata-containers/blob/3a7f9595b699a8ae25052736180de256ac10a5d1/tests/integration/kubernetes/tests_common.sh#L48-L51

they are defined and loaded implicitly, unfortunately.

Reply:
That's a shame. Thanks for checking